CurrencyControl/Model/CurrencyModel.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CurrencyControl.Model
+{
+    public class Currencies
```

Comment:
I don't think it should be public.

Remember about OOP principles. In this case encapsulation https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/tutorials/oop. There should be no public classes or members if they are not really needed or can be harmful. Perhaps this code cannot be harmful. Does the user need this class to work with CurrencyControl? If not - make it internal. 


We don't need useless public API.  Please check throughout the code and hide classes and members that should not be public in accordance with the OOP principles.

Reply:
Totally agree on usage of OOP principles In fact, l have made some changes in the whole solution and unfortunately, making certain components internal is not feasible, as they are utilized in property declarations bound to ComboBox elements. Let me know what you think thanks.